test/service.js

```diff
@@ -351,25 +351,26 @@ describe('Service', function() {
               name: 'app',
               networkBindings: [
                 {
-                  hostPort: 25002
-                }
-              ]
-            }
-          ]
+                  hostPort: 25002,
+                },
+              ],
+            },
+          ],
         },
       ]);
     });
 
     it('should report task health', function(done) {
-      AWS.mock('ECS', 'describeServices', function (params, cb){
+      AWS.mock('ECS', 'describeServices', function(params, cb) {
         cb(null, fixtures['newDeployment']);
       });
 
-      var service = new Service({clusterArn: 'cluster-9', serviceName: 'service-9'});
+      const service = new Service({clusterArn: 'cluster-9', serviceName: 'service-9'});
 
       service.on('updated', function() {
         expect(service.isTaskHealthy('arn::task:1')).to.equal(false);
         expect(service.isTaskHealthy('arn::task:2')).to.equal(true);
+        service.destroy();
```

Comment:
based on the later comment I think it makes sense to raise this to the top of the block.